io/zenoh-links/zenoh-link-quic/src/unicast.rs

```diff
@@ -156,6 +160,11 @@ impl LinkUnicastTrait for LinkUnicastQuic {
     fn is_streamed(&self) -> bool {
         true
     }
+
+    #[inline(always)]
+    fn get_auth_identifier(&self) -> LinkAuthId {
```

Comment:
Does it really need to be always cloned? Couldn't return `&LinkAuthId` and let to the caller the decision wether to call it or not?